RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Test.swift

```diff
@@ -27,12 +27,12 @@ extension CloudBackupClient: TestDependencyKey {
 	)
 
 	static let testValue = Self(
-		isCloudProfileSyncEnabled: unimplemented("\(Self.self).isCloudProfileSyncEnabled"),
+		isCloudProfileSyncEnabled: noop.isCloudProfileSyncEnabled,
```

Comment:
Because the tests where these clients are used fail when `unimplemented` is used

Reply:
Start the fail with the update to 1.17.0?

Reply:
Yes

Reply:
maybe we can use Sargon `.sample` values for `noop` in most clients? The fact that we throw is just because we did not have `sample` value for most types before Sargon.

Reply:
That’s a good idea, but it’ll take some time to go through all the noops, so I’m leaving it as a future improvement.